CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 
+### Add
+- Environment variable
+
```

Comment:
Please follow https://keepachangelog.com/en/1.1.0/ and add more complete description for this change.